blockchain/genesis_test.go

```diff
@@ -75,7 +75,7 @@ func TestHardCodedChainConfigUpdate(t *testing.T) {
 		},
 		{
 			name:       "baobab chainConfig update",
-			newHFBlock: big.NewInt(90909999),
+			newHFBlock: big.NewInt(79999999),
```

Comment:
AFAIK, 80000000 is london hardfork block number. How about commenting on this?
Also, if you add failure test cases (london compatible num < istanbul compatible) it would be nice.
If you add cypress test cases, it would be much nicer.

Reply:
@jeongkyun-oh Thanks, i applied at https://github.com/klaytn/klaytn/pull/1102/commits/13adb30131081da94198f8dfad442804d86b7112. I'll add another cypress test cases when adding cypress hard fork block numbers.